tools/ktablettl.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```

Comment:
I don't think we want to commit this

Reply:
Deleting the script file. I wasn't sure what the use of it was beyond creating the tutorial itself. Will remove.